### DISCUSSION TITLE
有人有買到GPT4的API KEY嗎？

### DISCUSSION THREAD

**Discussion by myforself** [yihong0618/bilingual_book_maker#315] [General]
網站上說是開放了，但付款那沒看見到有，是怎樣一回事呢…
雖然目前[bilingual_book_maker](https://github.com/yihong0618/bilingual_book_maker) GPT4 MODEL還不能用，但還是想準備一下…

**Comment by zitotw** [comment]
買plus付費會員就可以用gpt4 model
